python/running.md

```diff
@@ -0,0 +1,159 @@
+# Running Python
+
+To start using Python, we need access to the `python` program in a terminal. 
+This is available on lxplus, where we currently have access to version 2.7.13.
+
+```bash
+$ which python
+/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_88/Python/2.7.13/x86_64-slc6-gcc49-opt/bin/python
+$ python --version
+Python 2.7.13
+```
+
+We can add a few Python utilities to our environment by sourcing a setup file:
+
+```bash
+$ source /cvmfs/sft.cern.ch/lcg/views/LCG_88/$CMTCONFIG/setup.sh
```

Comment:
This doesn't work with `zsh` due to bugs in the LCG setup script (`PYTHONPATH` is set incorrectly but there could be other issues). It's probably best that we make sure the helpers are aware just get everyone to use bash.

Reply:
Ah, bummer. Yes, we should.